tools/ColorTool/ColorTool/Program.cs

```diff
@@ -200,6 +310,41 @@ static bool SetProperties(uint[] colorTable)
             }
             return success;
         }
+
+        static bool SetPropertiesWithVt(uint[] colorTable)
+        {
+            int STD_OUTPUT_HANDLE = -11;
+            IntPtr hOut = GetStdHandle(STD_OUTPUT_HANDLE);
+            uint originalMode;
+            uint requestedMode;
+            bool succeeded = GetConsoleMode(hOut, out originalMode);
+            if (succeeded)
+            {
+                requestedMode = originalMode | (uint)ConsoleAPI.ConsoleOutputModes.ENABLE_VIRTUAL_TERMINAL_PROCESSING;
+                SetConsoleMode(hOut, requestedMode);
+
+            }
+
+            for (int i = 0; i < 16; i++)
+            {
+                int vtIndex = VT_INDICIES[i];
+                uint rgb = colorTable[i];
+                string s = "\x1b]4;" + vtIndex + ";rgb:" + Rvalue(rgb).ToString("X") + "/"+ Gvalue(rgb).ToString("X") + "/"+ Bvalue(rgb).ToString("X") + "\x7";
```

Comment:
Oh boy. Please use a string formatting tool of some sort instead of building this up with concatenation.

I'd recommend using String Interpolation https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/string-interpolation

(e.g. string s = $"\x1b4];{vtIndex};rgb:{Rvalue(rgb),X}/{Gvalue(rgb),X}/{Bvalue(rgb),X}\x7";)